app/2.0/docs/devguide/custom-elements.md

```diff
@@ -306,16 +304,16 @@ MyMixin = function(superClass) {
         bar: {
           type: Object
         }
-      }
+      };
     }
 
     static get observers() {
-      return [ '_barChanged(bar.*)' ]
+      return [ ...superClass.observers, '_barChanged(bar.*)' ];
```

Comment:
This isn't necessary—double checked with Kevin & Steve and all of the superClass observers are invoked automatically. Demo here: https://jsbin.com/sidoco/5/edit?html,console,output